src/main/scala/DemoFortis.scala

```diff
@@ -98,6 +103,36 @@ object DemoFortis {
       }
     }
 
+    if (mode.contains("radio")) {
+      streamProvider.buildStream[RadioTranscription](ssc, streamRegistry("radio")) match {
+        case Some(stream) => stream
+          .map(radioTranscription => {
+            val language = Some(radioTranscription.language)
+            val keywords = keywordExtractor.extractKeywords(radioTranscription.text)
+            val analysis = Analysis(language = language, keywords = keywords)
+            val source = radioTranscription.radioUrl
+            AnalyzedItem(originalItem = radioTranscription, analysis = analysis, source = source)
+          })
+          .filter(analyzedRadio => {
+            supportedLanguages.contains(analyzedRadio.analysis.language.getOrElse(""))
+          })
+          .map(analyzedRadio => {
+            // sentiment detection
+            val text = analyzedRadio.originalItem.text
+            val language = analyzedRadio.analysis.language.getOrElse("")
+            val inferredSentiment = sentimentDetection.detectSentiment(text, language).map(List(_)).getOrElse(List())
```

Comment:
Heh yeah but I think the major offense is just that you can enumerate an Option. Since we're already doing that, toList doesn't really push the boundaries too much further IMO :)